css/css-flexbox_navigation/css/styles.css

```diff
@@ -29,3 +29,19 @@ ul {
   padding: 5px 14px;
   border-radius: 5px;
 }
+
+.navigation__menu {
+  display: flex;
+  justify-items: center;
```

Comment:
I think this property only applies to grid containers, probably also unnecessary.

Reply:
Talking about `justify-items: center;`